stress_test_model_loan_book.R

```diff
@@ -72,10 +71,8 @@ price_data_version <- cfg_st$price_data_version
 calculation_level <- cfg_st$calculation_level
 company_exclusion <- cfg_st$company_exclusion
 
-# set internal data location, should potentially be moved as this is directly in repo
-# data_location <- paste0(working_location,'data_updated_tool/')
-data_location <- file.path(working_location, data_path())
-# if this is moved out of repo, possibly use set_analysis_inputs_path()
+data_location <- file.path(get_st_data_path(), data_path())
+
 data_location <- ifelse(
```

Comment:
L76-80 seem like they will circumvent this new fanciness if this is run on the server, or more precisely when `twodii_internal == FALSE`.... it's that expected?

Reply:
In a way it is expected, but probably not necessary. Given the script will force any user to set up either an environment variable determining the data location or an option to do that, there is not really any point in allowing users to override that via the config I guess. Best case it's redundant, worst case it leads to errors or unexpected behaviour/results.. Looks like it's time to remove this option

Reply:
so this data_location_ext stuff is removed now. doing that even allowed me to get rid of some confusing and unnecessary paths that I used to set up for nothing in the web tool scripts. should make it a bit easier to figure out the integration once the time comes.